### DISCUSSION TITLE
Windows 11 start button customization

### DISCUSSION THREAD

**Discussion by rlnt** [farag2/Sophia-Script-for-Windows#286] [Ideas]
Hello! Huge fan of the sophia script for a long time already.
I recently switched to Windows 11 and it would be awesome if there was a tweak in the script to directly open the "all apps" list when clicking the Windows 11 start button.
I am not a huge fan of the little menu which pops up because it will force you to have an additional step before you can see all apps. It's not even possible to disable the recommended app section in the start menu which is really annoying so this option would be really handy.

**Comment by farag2** [comment]
Thabks for you kind words. Unfortunately, MS has blocked the access to porgrammatically configuring the Start menu like I did on 10. What's for you request, there is no way to  do this. The only thin I may advice is to wait for Start11 release: https://twitter.com/bdsams/status/1451622654591520771

**Reply by rlnt** [reply]
I've seen it here https://github.com/valinet/ExplorerPatcher
I thought that could be achieved with the script as well but they probably patch a lot of stuff to do this.

Thanks for the info!

**Reply by farag2** [reply]
Good point, but I won't let installing any 3rd party, uknown DLLs in my module. It may be cause of OS instability, even BSoDs. Today it works, tomorrow it may be flagged as a malicious code. So I STRONGLY advice not to use such hacks.
